.circleci/config.yml

```diff
@@ -15,7 +15,7 @@ experimental:
 templates:
   job_template: &job_template
     docker:
-      - image: gcr.io/datadoghq/agent-circleci-runner:v37150619-8250a4fc
+      - image: gcr.io/datadoghq/agent-circleci-runner_test_only:v38257968-842d2882
```

Comment:
Is the renaming `agent-circleci-runner` to `agent-circleci-runner_test_only` expected?

Reply:
It's named `agent-circleci-runner_test_only` because the image on the `datadog-agent-buildimages` is still in a PR branch, not merged on the main branch.